ui/app/models/pki/pki-role-engine.js

```diff
@@ -140,25 +273,44 @@ export default class PkiRoleEngineModel extends Model {
         {
           'Domain handling': [
             'allowedDomains',
-            'allowedDomainTemplate',
+            'allowedDomainsTemplate',
             'allowBareDomains',
             'allowSubdomains',
             'allowGlobDomains',
             'allowWildcardCertificates',
-            'allowLocalhost',
+            'allowLocalhost', // default: true (returned true by OpenApi)
             'allowAnyName',
-            'enforceHostnames',
+            'enforceHostnames', // default: true (returned true by OpenApi)
+            'moreInfo', // shows as helperText with icon at bottom of the options box.
           ],
         },
         {
-          'Key parameters': ['keyType', 'keyBits', 'signatureBits'],
+          'Key parameters': ['keyParametersLabel', 'keyType', 'keyBits', 'signatureBits'],
         },
         {
           'Key usage': [
-            'DigitalSignature', // ARG TODO: capitalized in the docs, but should confirm
+            'keyUsageLabel',
+            'DigitalSignature', // keeping these values capitalized to distinguish them from actual API params. The serializer changes them to an arrayList.
+            'ContentCommitment',
+            'CrlSign',
             'KeyAgreement',
+            'DataEncipherment',
+            'EncipherOnly',
             'KeyEncipherment',
-            'extKeyUsage', // ARG TODO: takes a list, but we have these as checkboxes from the options on the golang site: https://pkg.go.dev/crypto/x509#ExtKeyUsage
+            'CertSign',
+            'DecipherOnly',
+            'extKeyUsageLabel',
+            'Any',
+            'EmailProtection',
+            'TimeStamping',
+            'ServerAuth',
+            'IpsecEndSystem',
+            'OcspSigning',
+            'ClientAuth',
+            'IpsecTunnel',
+            'IpsecUser',
+            'CodesSigning',
+            'extKeyUsageOids',
```

Comment:
This could potentially be replaced with `keyUsage` and `extKeyUsage` if those were the attrs instead of each individual available values for those arrays. Those attrs could then yield and the checkboxes could be handled separately.